### ISSUE TITLE
Unsupported “hardwareModel” trying to start ventura-base

### ISSUE THREAD

**Issue by damonbreeden** [cirruslabs/tart#489]
Hi, trying to start a new VM today, I followed the getting started instructions but getting an error
i ran `brew install tart` before starting, but `tart --version` looks off
`tart run monterey-base` works fine
```
damonbreeden@Damons-MacBook-Pro-2 ~ % tart --version
SNAPSHOT
damonbreeden@Damons-MacBook-Pro-2 ~ % tart list
Source Name                                                                                                           Size
local  m1_test                                                                                                        18  
local  monterey-base                                                                                                  30  
local  monterey-vanilla                                                                                               18  
oci    ghcr.io/cirruslabs/macos-monterey-base:latest                                                                  30  
oci    ghcr.io/cirruslabs/macos-monterey-base@sha256:4a4595e47d48311aed04445191b90d62a76390ea8b32866e35547d8856bfae44 30  
oci    ghcr.io/cirruslabs/macos-monterey-base@sha256:b9a3f802000a4666190c032849dc4aedc88ccb8c6818feee6f1e0cf241ccce26 30
damonbreeden@Damons-MacBook-Pro-2 ~ % tart clone ghcr.io/cirruslabs/macos-ventura-base:latest ventura-base
pulling manifest...
pulling disk (16.3 GB compressed)...
100%
pulling NVRAM...
damonbreeden@Damons-MacBook-Pro-2 ~ % tart run ventura-base
Error: Invalid virtual machine configuration. Unsupported “hardwareModel”.
damonbreeden@Damons-MacBook-Pro-2 ~ % 
damonbreeden@Damons-MacBook-Pro-2 ~ % system_profiler SPSoftwareDataType SPHardwareDataType
Software:

    System Software Overview:

      System Version: macOS 12.4 (21F79)
      Kernel Version: Darwin 21.5.0
      Boot Volume: Macintosh HD
      Boot Mode: Normal
      Computer Name: Damon’s MacBook Pro (2)
      User Name: Damon Breeden (damonbreeden)
      Secure Virtual Memory: Enabled
      System Integrity Protection: Enabled
      Time since boot: 37 days 22:33

Hardware:

    Hardware Overview:

      Model Name: MacBook Pro
      Model Identifier: MacBookPro17,1
      Chip: Apple M1
      Total Number of Cores: 8 (4 performance and 4 efficiency)
      Memory: 16 GB
      System Firmware Version: 7459.121.3
      OS Loader Version: 7459.121.3
      Serial Number (system): <redacted>
      Hardware UUID: <redacted>
      Provisioning UDID: <redacted>
      Activation Lock Status: Enabled

damonbreeden@Damons-MacBook-Pro-2 ~ % 
```

**Comment by edigaryev** [comment]
Looks like a duplicate of https://github.com/cirruslabs/tart/issues/469.

**Comment by damonbreeden** [comment]
thanks, i didn't look through the closed issues for similar issues

i followed the instructions there, that works (with `ghcr.io/cirruslabs/macos-ventura-vanilla:13.2.1`)

maybe the images can support some kind of tag that tells you what the minimum host version is?

**Comment by edigaryev** [comment]
> maybe the images can support some kind of tag that tells you what the minimum host version is?

The `hardwareModel` contained in `config.json` for macOS VM images actually contains all that:

```
% cat ~/.tart/vms/<VM name>/config.json | jq -r '.hardwareModel' | base64 -d | plutil -p - 
{
  "DataRepresentationVersion" => 1
  "MinimumSupportedOS" => [
    0 => 13
    1 => 0
    2 => 0
  ]
  "PlatformVersion" => 2
}

```

However, Apple decided to not expose these fields for some reason in the `Virtualization.Framework` API nor to emit an error message providing details as to what exactly in `hardwareModel` that is unsupported, instead of just `Unsupported “hardwareModel”` 🤷

**Comment by fkorotkov** [comment]
I think we can at least improve the message and suggest to update host OS version. I've created #491 to improve it.
